docs/Documentation/Applications/vasp.md

```diff
@@ -32,62 +42,93 @@ NREL offers modules for VASP 5 and VASP 6 on CPUs as well as GPUs on certain sys
 | VASP 6      |       X       |       X       |       X       |
 | VASP 6 GPU  |       X       |       X       |       X       |
 
-Each VASP module provides three executables where the correct one should be chosen for the type of job:
-
-1. `vasp_std` is for general k-point meshes with collinear spins
-
-2. `vasp_ncl` is for general k-point meshes with non-collinear spins
-
-3. `vasp_gam` is for Gamma-point-only calculations
-
-NREL also offers support for additional functionalities such as [transition state theory tools from University of Texas-Austin](http://theory.cm.utexas.edu/vtsttools/), [implicit solvation models from the University of Florida](http://vaspsol.mse.ufl.edu/), and [BEEF-vdw functionals](https://github.com/vossjo/libbeef). Please contact [HPC-Help](mailto:hpc-help@nrel.gov) if a functionality you need is not present in one of our builds.
 
 ## VASP on Kestrel
 
 ### Running Using Modules
 
 #### CPU
 
-There are modules for CPU builds of VASP 5 and VASP 6 each with solvation, transition state tools, and BEEF-vdW functionals. These modules can be loaded with ```module load vasp/<version>```.
+There are several modules for CPU builds of VASP 5 and VASP 6. As of 08/09/2024 we have released new modules for VASP on Kestrel CPUs: 
+
+```
+------------ /nopt/nrel/apps/cpu_stack/modules/default/application -------------
+   #new modules:
+   vasp/5.4.4+tpc     vasp/6.3.2_openMP+tpc    vasp/6.4.2_openMP+tpc
+   vasp/5.4.4_base    vasp/6.3.2_openMP        vasp/6.4.2_openMP
+   
+   #legacy modules will be removed during next system time:
+   vasp/5.4.4         vasp/6.3.2               vasp/6.4.2            (D)
+```
+
+ What’s new:
```

Comment:
```suggestion
 What’s new: 
 
```

Reply:
Need the whitespace suggested here in order for markdown to render the bullets correctly